JacquardToolkit/JSConstants.swift

```diff
@@ -81,5 +81,18 @@ struct JSConstants {
         }
         
     }
+    
+    struct JSURLs {
```

Comment:
Less bundled app assets by way of online hosted gifs. Kinda like it. Just some things to consider:
- this prevents the toolkit from being completely offline/internet independent
- imgur personal videos might/probably are deleted after some time off the website, probably best to host this elsewhere if you still want to go the online route?